src/server/pfs/server/driver_v2.go

```diff
@@ -206,7 +205,7 @@ func (d *driver) getFilesConditional(ctx context.Context, repo, commit, glob str
 }
 
 func matchFunc(glob string) (func(string) bool, error) {
-	// (bryce) this is a little weird, but it prevents the parent directory from being matched (i.e. /*).
+	// TODO Need to do a review of the globbing functionality.
```

Comment:
Make sure to do this one carefully, we've had some really subtle hard to track down bugs come from globbing semantics.